lib/web_ui/lib/src/engine/skwasm/skwasm_impl/shaders.dart

```diff
@@ -240,7 +280,20 @@ class SkwasmFragmentShader extends SkwasmShader implements ui.FragmentShader {
 
   @override
   void setImageSampler(int index, ui.Image image) {
-    // TODO(jacksongardner): implement this when images are implemented
+    final SkwasmImageShader shader = SkwasmImageShader.imageShader(
+      image as SkwasmImage,
+      ui.TileMode.clamp,
+      ui.TileMode.clamp,
+      toMatrix64(Matrix4.identity().storage),
```

Comment:
We should probably cache this.

Reply:
Actually, I think I should just make this optional and not pass the data at all for the identity case.